File: api/src/main/java/utn/dacs/ms/backend/model/entity/Partido.java
package utn.dacs.ms.backend.model.entity;

import javax.persistence.*;


import lombok.Data;

import java.util.Date;
import java.util.List;

@Data
@Entity
public class Partido {

    @Id
    private Integer id;
    private String equipo1;
    private String equipo2;
    private Integer puntosEquipo1;
    private Integer puntosEquipo2;
    @OneToMany(mappedBy = "partido")
    private List<EstadisticasPartido> estadisticasPartido;

}

File: web/src/app/models/partido.ts
export interface Partido {
    id: number;
    equipo1: {
        id: string;
        nombre: string;
        logo: string;
    };
    equipo2: {
        id: string;
        nombre: string;
        logo: string;
    };
    puntosEquipo1: number;
    puntosEquipo2: number;
  }
